bican.tfvars

```
@@ -0,0 +1,6 @@
+singleuser_image_repo = "dandiarchive/dandihub"
+region = "us-east-1"
```

Currently most DANDI and LINC infrastructure are in `us-east-2`, since the DANDI OpenData program is in `us-east-2`.  If BICAN does not have data in `us-east-1`, I would suggest that we keep all resources in `us-east-2` so its easier for our team to keep track of the various components that are spun up manually and with Terraform.

```suggestion
region = "us-east-2"
```